functional_tests/test_validator.py

```diff
@@ -18,8 +18,6 @@ def _validator(file_path, for_appversions=None, overrides=None):
     import validator
     import validator.constants
     js = os.environ.get('SPIDERMONKEY_INSTALLATION', 'js')
-    scripting.SPIDERMONKEY_INSTALLATION = js
-    validator.constants.SPIDERMONKEY_INSTALLATION = js
```

Comment:
`SPIDERMONKEY_INSTALLATION` isn't used in validator code at all. So it made no sense to me to set it.